tmt/steps/report/junit/templates/subresults.xml.j2

```diff
@@ -0,0 +1,80 @@
+{% extends "_base.xml.j2" %}
+
+{#
+    This flavor changes the level of `<testsuite>` and `<testcase>` tags. The
+    `tmt.Result` becomes ``<testsuite>`` instead of ``testcase`` and
+    ``<testcase>`` tags become ``tmt.SubResult``.
+#}
+
+{% block content %}
+    <testsuites disabled="0" errors="{{ RESULTS.errored | length }}" failures="{{ RESULTS.failed | length }}" tests="{{ RESULTS | length }}" time="{{ RESULTS.duration }}">
+
+    {% block testsuites %}
+        {% for result in RESULTS %}
+            {% set main_log = result.log | first | read_log %}
```

Comment:
The logs must not be saved into the memory/variable. It must be handled same way as in: https://github.com/teemtee/tmt/pull/3382